.drone.yml

```diff
@@ -0,0 +1,227 @@
+workspace:
+  base: /var/www/owncloud
+  path: apps/password_policy
+
+branches: [master, release*, release/*]
+
+pipeline:
+  install-server:
+    image: owncloudci/core
+    version: ${OC_VERSION}
+    pull: true
+    db_type: ${DB_TYPE}
+    db_name: ${DB_NAME}
+    db_host: ${DB_TYPE}
+    db_username: autotest
+    db_password: owncloud
+    exclude: apps/password_policy
+    when:
+      matrix:
+        NEED_SERVER: true
+
+  install-app:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    commands:
+      - cd /var/www/owncloud/
+      - php occ a:l
+      - php occ a:e password_policy
+      - php occ a:e testing
+      - php occ a:l
+      - php occ config:system:set trusted_domains 1 --value=owncloud
+      - php occ log:manage --level 0
+    when:
+      matrix:
+        NEED_INSTALL_APP: true
+
+  fix-permissions:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    commands:
+    - chown www-data /var/www/owncloud -R
```

Comment:
WHY? we do not have any job that requires `www-data` as user

Reply:
not yet, acceptance tests run as `www-data`

Reply:
I wonder why we need to do that, in general. The acceptance tests should be hitting APIs and logging in to the webUI. So they should run as "any OS client user" that gives credentials to the API or logs in to the webUI.
@individual-it why is it so?

Reply:
Running the acceptance test script works fine without it being ``www-data`` - as it should.
The only trick sometimes will be when ``chown www-data /var/www/owncloud -R`` has happened. If that happens to take read access away for ``/var/www/owncloud/tests/acceptance`` then there would be a problem. That would be just because the acceptance test scripts are "hiding" inside the server code. Some day that may get separated and thus avoid even this possible issue.

See PR #112 that removes the excess ``.drone.yml`` code.